firebaseai/src/GenerateContentResponse.cs

```diff
@@ -65,7 +81,8 @@ public string Text {
   /// </summary>
   public IReadOnlyList<ModelContent.FunctionCallPart> FunctionCalls {
```

Comment:
Do you also need to add the same logic for getting InlineDataPart?

Reply:
There currently isn't a helper function for that. I guess I could add one...